OpenRA.Game/Game.cs

```diff
@@ -410,6 +422,14 @@ static void Initialize(Arguments args)
 			InitializeMod(modID, args);
 		}
 
+		static Size GetResolution(GraphicSettings graphicsSettings)
+		{
+			var size = (graphicsSettings.Mode == WindowMode.Windowed)
+				? graphicsSettings.WindowedSize
+				: graphicsSettings.FullscreenSize;
+			return new Size(size.X, size.Y);
```

Comment:
Style nit: I know you're just moving around existing code here, but please put a newline between the return and var for readability while you're at it.